ODLA/platforms/tensorrt/odla_tensorrt.cc

```diff
@@ -1434,7 +1433,8 @@ odla_value odla_InstanceNormalization(
   auto plugin = creator->createPlugin(plugin_name, &plugin_data);
   auto norm = g_comp->network->addPluginV2(
       &inputs[0], static_cast<int>(inputs.size()), *plugin);
-  return CreateValue(norm->getOutput(0), input->type, value_id);
+  auto v = CreateValue(norm->getOutput(0), input->type, nullptr);
+  return odla_Add(odla_Mul(v, scale, nullptr), offset, value_id);
```

Comment:
Is the calculation correct? Tensors of "scale" and "offset" are used twice, once in TensorRT plugin and once in the final step mul+add?

Reply:
Oh, looks like the original code is correct. 
But the test always fails as it can't find the "instance norm" in plugin. Do you know why?